src/app/modules/submission/email-submission/email-submission.controller.ts

```diff
@@ -140,6 +144,8 @@ export const handleEmailSubmission: RequestHandler<
         .status(statusCode)
         .json({ message: errorMessage, spcpSubmissionFailure: true })
     }
+    // MyInfo access token is single-use, so clear it
+    res.clearCookie(MYINFO_COOKIE_NAME, MYINFO_COOKIE_OPTIONS)
```

Comment:
move this downwards to after submission is saved in the database

Reply:
moved to after admin email is sent, right before returning response